Phishing campaign & ML - added documentation regarding XSIAM (#33414)
demisto/content: Packs/Phishing/ReleaseNotes/3_6_13.md

#### Playbooks

##### Phishing - Generic v3

> - Updated the documentation to state what platforms support the machine-learning content used in the playbook.
> 
> ##### Phishing - Machine Learning Analysis
> 
> - Updated the documentation to state what platforms support the machine-learning content used in the playbook.

Reviewer: ```suggestion
Updated the documentation to state what platforms support the machine-learning content used in the playbook.

##### Phishing - Machine Learning Analysis

Updated the documentation to state what platforms support the machine-learning content used in the playbook.
```

#### Playbooks

##### Phishing - Generic v3

- Updated the documentation to state what platforms support the machine-learning content used in the playbook.
- The playbook will now skip the *Phishing - Machine Learning Analysis* playbook when it is unavailable.

##### Phishing - Machine Learning Analysis

- Updated the documentation to state what platforms support the machine-learning content used in the playbook.

> - The playbook is now unavailable in XSIAM due to the automations used in it.

Reviewer 2: ```suggestion
- The playbook is now available only for the XSOAR marketplace.
```

Author: Better
